aws/resource_aws_route53_health_check_test.go

```diff
@@ -690,3 +715,19 @@ resource "aws_route53_health_check" "test" {
 }
 `, disabled)
 }
+
+const testAccRoute53HealthCheckConfigRoutingControlArn = `
+resource "aws_route53recoverycontrolconfig_cluster" "test" {
+  name = "tf-acc-test-cluster"
+}
+
+resource "aws_route53recoverycontrolconfig_routing_control" "test" {
+  name        = "tf-acc-test"
+  cluster_arn = aws_route53recoverycontrolconfig_cluster.test.arn
+}
+
+resource "aws_route53_health_check" "test" {
+  type                = "RECOVERY_CONTROL"
+  routing_control_arn = aws_route53recoverycontrolconfig_routing_control.test.arn
+}
+`
```

Comment:
you'll also want to randomly generate these names e.g.
```suggestion
func testAccRoute53HealthCheckConfigRoutingControlArn (rName string) string {
    return fmt.Sprintf(`
resource "aws_route53recoverycontrolconfig_cluster" "test" {
  name = %[1]q
}

resource "aws_route53recoverycontrolconfig_routing_control" "test" {
  name        = %[1]q
  cluster_arn = aws_route53recoverycontrolconfig_cluster.test.arn
}

resource "aws_route53_health_check" "test" {
  type                = "RECOVERY_CONTROL"
  routing_control_arn = aws_route53recoverycontrolconfig_routing_control.test.arn
}
`, rName)
}
```

Reply:
done